akkeeper/src/main/scala/akkeeper/master/route/BaseController.scala

```diff
@@ -28,6 +28,7 @@ import scala.collection.mutable
 import scala.reflect.ClassTag
 
 trait BaseController extends Directives with SprayJsonSupport with CommonApiJsonProtocol {
+  val service: ActorRef
```

Comment:
I don't think that this member should be publicly visible/available. Also I don't recommend to use abstract `val` since it leads to very strict limitations on how it can be overridden in descendant classes. I'd suggest to always use abstract `def` instead, because it can be overridden with pretty much anything (concrete `def`, `val`, `lazy val`).